features/sqlup-mode.el.feature

```diff
@@ -21,3 +21,40 @@ Feature: Upcasing as I type
     And I insert "addslots"
     And I type " "
     Then I should see "ADDSLOTS "
+
+ Scenario: Upcase a normal SQL keyword in a postgres-execute eval string
+    Given I turn on sqlup-mode
+    And I start an action chain
+    And I press "M-x"
+    And I type "sql-highlight-postgres-keywords"
+    And I execute the action chain
+    And I insert "execute"
+    And I type " "
+    And I insert "'select"
+    And I type " "
+    Then I should see "EXECUTE 'SELECT "
+
+ Scenario: Upcase a normal SQL keyword in a postgres-format eval string
+    Given I turn on sqlup-mode
+    And I start an action chain
+    And I press "M-x"
+    And I type "sql-highlight-postgres-keywords"
+    And I execute the action chain
+    And I insert "execute"
+    And I type " "
+    And I insert "format("
+    And I insert "'select"
+    And I type " "
+    Then I should see "EXECUTE format('SELECT "
```

Comment:
Format isn't a keyword but a function, so it usually isn't written in caps, AFAIK. Of course we can decide to do otherwise, there's not clear standard here.